python/ribasim/ribasim/models.py

```diff
@@ -161,6 +168,7 @@ class BasinStatic(BaseModel):
 class Root(BaseModel):
     DiscreteControlLogic: Optional[DiscreteControlLogic] = None
     Edge: Optional[Edge] = None
+    FlowBoundaryTime: Optional[FlowBoundaryTime] = None
```

Comment:
Nit, but it's easier to read if this is next to the FlowBoundaryStatic. Perhaps we should sort all these lines alphabetically (your editor can do that).